lib/footnotes.js

```diff
@@ -24,24 +24,22 @@ function setRef(link)
 
 function processPage(page)
 {
-  var options = this.config.options
+  if(this.output.name !== 'ebook') return page
 
-  if(options.generator !== 'pdf') return page
-
-  var language = intl[options.language || 'en']
+  var i10nPrefix = intl[this.config.get('language')] || intl['en']
```

Comment:
Then here we have two options, deprecate to the english language, or show an error and try to improve the module to support all available languages...